var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -56,11 +60,13 @@ class Hwloc(AutotoolsPackage):
 
     depends_on('pkgconfig', type='build')
 
+    depends_on('cuda', when='+nvml')
     depends_on('cuda', when='+cuda')
+    depends_on('opengl', when='+gl')
```

Comment:
this should just read `depends_on('gl', when='+gl')`.

In Spack, `gl` is a virtual package and e.g. `opengl` provides it.

Reply:
Ok no problem.